src/lib/settings/initialSettings.tsx

```diff
@@ -0,0 +1,383 @@
+                v ===
+                settingsContext.modeling.defaultUnit[
+                  cmdContext.argumentsToSubmit.level as SettingsLevel
+                ],
+            })),
+        },
+      }),
+      /**
+       * The controls for how to navigate the 3D view
+       */
+      mouseControls: new Setting<CameraSystem>({
+        defaultValue: 'KittyCAD',
+        description: 'The controls for how to navigate the 3D view',
+        validate: (v) => cameraSystems.includes(v as CameraSystem),
+        hideOnLevel: 'project',
+        commandConfig: {
+          inputType: 'options',
+          defaultValueFromContext: (context) =>
+            context.modeling.mouseControls.current,
+          options: (cmdContext, settingsContext) =>
+            Object.values(cameraSystems).map((v) => ({
+              name: v,
+              value: v,
+              isCurrent:
+                v ===
+                settingsContext.modeling.mouseControls[
+                  cmdContext.argumentsToSubmit.level as SettingsLevel
+                ],
+            })),
+        },
+        Component: ({ value, onChange }) => (
+          <>
+            <select
+              id="camera-controls"
+              className="block w-full px-3 py-1 bg-transparent border border-chalkboard-30"
+              value={value}
+              onChange={onChange}
+            >
+              {cameraSystems.map((program) => (
+                <option key={program} value={program}>
+                  {program}
+                </option>
+              ))}
+            </select>
+            <ul className="mx-0 my-2 flex flex-col gap-2 text-sm">
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Pan</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].pan.description}
+                </p>
+              </li>
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Zoom</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].zoom.description}
+                </p>
+              </li>
+              <li className="grid grid-cols-4 gap-1">
+                <strong>Rotate</strong>
+                <p className="col-span-3 leading-tight">
+                  {cameraMouseDragGuards[value].rotate.description}
+                </p>
+              </li>
+            </ul>
+          </>
+        ),
+      }),
+      /**
+       * Whether to show the debug panel, which lets you see
+       * various states of the app to aid in development
+       */
+      showDebugPanel: new Setting<boolean>({
+        defaultValue: false,
+        description: 'Whether to show the debug panel, a development tool',
+        validate: (v) => typeof v === 'boolean',
+        commandConfig: {
+          inputType: 'boolean',
+        },
+      }),
+      /**
+       * TODO: This setting is not yet implemented.
```

Comment:
If we do add this to settings, I don't think it should be a boolean, but instead similar to the theme `'system' | 'on' | 'off'`, and have it default to `system` since that's what we're currently using for animations.

Reply:
Ah right that makes perfect sense